app/views/finders/_result_count_generic.mustache

```diff
@@ -0,0 +1,8 @@
+{{#any_filters_applied}}
+<p class="result-info">
+  <span class="result-count">
+    {{total}}
+  </span>
+  {{{generic_description}}}
```

Comment:
Is this meant to have three `{{{`?

Reply:
I guess technically not since it's just a string but in case we add HTML this will allow it to appear correctly.